test/e2e/fixtures.go

```diff
@@ -46,6 +46,18 @@ func skipIfNumNodesLessThan(tb testing.TB, required int) {
 	}
 }
 
+func skipIfNotIPv4Cluster(tb testing.TB) {
+	if clusterInfo.podV4NetworkCIDR == "" {
+		tb.Skipf("Skipping test as it requires in IPv4 address but the IPv4 network CIDR is not set")
+	}
+}
+
+func skipIfIPv6Cluster(tb testing.TB) {
+	if clusterInfo.podV6NetworkCIDR != "" {
+		tb.Skipf("Skipping test as it requires in IPv4 address but the IPv4 network CIDR is not set")
```

Comment:
I feel like the log message is misleading, as it's the same as for `skipIfNotIPv4Cluster`. It seems that you want to skip the test if the cluster support IPv6.

Reply:
Updated.